### ISSUE TITLE
Problem: [/moderator] UX/UI isn't very good

### ISSUE THREAD

**Issue by AshHimself** [Blockrazor/blockrazor#908]
Problem: UX is not very good and hard to use

Solution: Bootwatchify & improve UX.

- Group/filter approval types
- bootwatchify Moderation Queue menu items

@gazhayes - I don't see removeUserCredit used at all for when items are flagged? Should this be in the workflow. I can jump on this screen tonight if need be as I am currently not on anything.

**Comment by gazhayes** [comment]
Actually just hold off on this one, I'll ping you about it later. Going to log a few structural problems first.

**Comment by AshHimself** [comment]
@gazhayes any update on this one mate?

**Comment by gazhayes** [comment]
Ah no. But I don't want to hold it up any longer.

How do you think that page should be laid out? It's kind of annoying to have to click into everything. I was thinking about maybe concertina'd left menu for moderators so they can easily see everything there without going back to the main moderator page.

**Comment by AshHimself** [comment]
Taking this, will respond what I think we should do before starting.

**Comment by gazhayes** [comment]
FYI by 'concertina'd side menu ' I mean collapsible sub-menus on the existing side menu.

**Comment by AshHimself** [comment]
Something like https://www.codeply.com/go/8ESO56VMns/bootstrap-4-sidebar-menu-(toggleable)
Then we can add as sub menu items to reduce the amounts of clicks required.

- See flagged users
- See questions
- See flagged hashpower
- See applogs
- See solved problems
- Pardon suspended users
- Flagged comments/red flags/features

As for image approvals, all image approvals seem to be bunched all onto the main page. That could be split as well, or keep them all on the same page with a filter and easily display what each row is.

Thoughts?

**Comment by gazhayes** [comment]
That menu looks like the way to go.

You should probably make a new issue just for menu restructuring and do it in there and then come back to this one after.

**Comment by gazhayes** [comment]
> all image approvals seem to be bunched all onto the main page.

I'd say keep them together, for now. Though more structural work needs to be done on that, for example images need to be embiggenable and things like community url need to be show too.

**Comment by AshHimself** [comment]
@anbud - do we still need the ELO shortcuts?

**Comment by anbud** [comment]
@AshHimself, it's done in a cron job every 10 minutes I believe, so, no, I left them there just for testing purposes, but I mostly call methods from console anyways. Feel free to remove them.

**Comment by AshHimself** [comment]
Removed.. legend!

**Comment by AshHimself** [comment]
@gazhayes - Do we have a view at present to view a currencies attributes before approving? I don't think we do from initial look.

**Comment by gazhayes** [comment]
> whats the progress on this one? there are number of moderator pages that will to be bootwatchified

These are the ones that should **not** be done because I'm making structural changes:
Flagged users
rating questions
application logs
solved problems

**Comment by AshHimself** [comment]
Ok I have just done an initial push then which clean up a lot of things. I'll move into the more functional components of it. e.g. reviewing what people are actually approving :)
